configure

```diff
@@ -116,9 +116,7 @@ check_go_env() {
 
 cd ${0%/*}
 
-if [ ! -f .env ]; then
-  rm .env
-fi
+rm -fv .env
```

Comment:
Could you quickly explain the purpose of this change? What's the difference in behaviour?

Reply:
So, the original logic is "if '.env' is not a file, remove it". If the file doesn't exist, the test passes and then rm returns an error message as the file doesn't exist. I _believe_ the intent was to remove the file if it exists since it's set by `./configure`. I updated it to simply and reflect that by adding the `-f` which will silently ignore missing files.

I could be wrong, but that was my understanding in reading `configure`.

Reply:
Sounds good